src/Windows/dll/evtx_logging.h

```diff
@@ -0,0 +1,87 @@
+#ifndef _EXAMPLE_EVENT_LOG_MESSAGE_FILE_H_
+#define _EXAMPLE_EVENT_LOG_MESSAGE_FILE_H_
```

Comment:
I don't see the message file updates to the resource file for the DLL to add these message resources, did I miss it?